interface/wx/print.h

```diff
@@ -705,23 +705,39 @@ class wxPrintout : public wxObject
         and maximum page values that the user can select, and the required page range to
         be printed.
 
-        If the user chose to print only selected pages in the MSW printing
-        dialog, then @a pageFrom and @a pageTo are used to limit the page range
-        and IsPageSelected() is called later to query whether the page is
-        selected and so should be printed.
-
-        If the user chose to print the current page, then @a pageFrom and
-        @a pageTo should be both set to the current page number.
-
         By default this returns (1, 32000) for the page minimum and maximum values, and
         (1, 1) for the required page range.
 
         @a minPage must be greater than zero and @a maxPage must be greater
         than @a minPage.
+
+        @deprecated
+        The newer method that support multiple ranges should be used instead.
```

Comment:
This is not quite right, as this function is not formally deprecated. I'd probably say something like

```suggestion
        This function allows to indicate only a single range of pages to print,
        consider overriding GetPageInfoRanges() to return multiple ranges.
```